src/scripts/audit/tool.sh

```diff
@@ -0,0 +1,305 @@
+#!/bin/bash
+# Copyright 2020, Oracle Corporation and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+#set -x
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/utility.sh
```

Comment:
For the benefit of Joe's customers, we probably need to keep this script from referencing others so that customers have less to review. Remember that some of the customers using this script will not be using the operator and so additional dependencies will be suspicious.

Reply:
ok, yes. I had considered this but later thought that extracting common library type functions and utilities to a separate script might make the main script (`tool.sh`) more readable and easier to review. This might also be helpful if we need to add more library type functions in future. Please note that utility.sh doesn't have a dependency on operator, it'll resides in same directory as `tool.sh` and will be shipped with the zip file containing audit tool script and artifacts file. 

I can consolidate it into a single script as you recommended. I just wanted to discuss before merging two scripts into one.

Reply:
As discussed in meeting, utility.sh is part of this PR and will be shipped with tool.sh. I can rename it to helper.sh or something else if that helps avoiding the confusion.